packages/viem/src/test/e2e/interop.spec.ts

```diff
@@ -1,84 +1,57 @@
-import {
-  createPublicClient,
-  createWalletClient,
-  encodeFunctionData,
-  http,
-  keccak256,
-} from 'viem'
+import { encodeFunctionData } from 'viem'
 import { describe, expect, it } from 'vitest'
 
-import { supersimL2A, supersimL2B } from '@/chains/supersim.js'
-import { publicActionsL2 } from '@/decorators/publicL2.js'
-import { walletActionsL2 } from '@/decorators/walletL2.js'
-import { testAccount } from '@/test/clients.js'
+import { supersimL2B } from '@/chains/supersim.js'
+import {
+  publicClientA,
+  publicClientB,
+  testAccount,
+  walletClientA,
+  walletClientB,
+} from '@/test/clients.js'
 import { ticTacToeABI, ticTacToeAddress } from '@/test/setupTicTacToe.js'
-import { decodeExecutingMessage } from '@/utils/decodeExecutingMessage.js'
-import { extractMessageIdentifierFromLogs } from '@/utils/extractMessageIdentifierFromLogs.js'
-
-const RPC_URL_A = supersimL2A.rpcUrls.default.http[0]
-const RPC_URL_B = supersimL2B.rpcUrls.default.http[0]
-
-const publicClientA = createPublicClient({
-  chain: supersimL2A,
-  transport: http(RPC_URL_A),
-}).extend(publicActionsL2())
-
-const publicClientB = createPublicClient({
-  chain: supersimL2B,
-  transport: http(RPC_URL_B),
-}).extend(publicActionsL2())
-
-export const walletClientA = createWalletClient({
-  account: testAccount,
-  chain: supersimL2A,
-  transport: http(RPC_URL_A),
-}).extend(walletActionsL2())
-
-export const walletClientB = createWalletClient({
-  account: testAccount,
-  chain: supersimL2B,
-  transport: http(RPC_URL_B),
-}).extend(walletActionsL2())
+import {
+  createInteropSentL2ToL2Messages,
+  decodeRelayedL2ToL2Messages,
+} from '@/utils/l2ToL2CrossDomainMessenger.js'
 
 describe('Generic Interop Flow', () => {
   it('should send and execute cross chain message', async () => {
```

Comment:
i'm not sure why this test is failing in CI i'm starting to think theres some concurrency issue happening we can add `it.skip` to this so we can work on getting the PR merged in and come back to this and figure out whats happening

Reply:
confusing this is that this I think this is an error in the backend meaning it's hitting anvil